README.md

```diff
@@ -377,15 +378,7 @@ Julia uses a custom fork of libuv. It is a small dependency, and can be safely b
 
 As a high-performance numerical language, Julia should be linked to a multi-threaded BLAS and LAPACK, such as OpenBLAS or ATLAS, which will provide much better performance than the reference `libblas` implementations which may be default on some systems.
 
-### SuiteSparse
-
-SuiteSparse is a special case, since it is typically only installed as a static library, while `USE_SYSTEM_SUITESPARSE=1` requires that it is a shared library. Running the script `contrib/repackage_system_suitesparse4.make` will copy your static system SuiteSparse installation into the shared library format required by Julia. `make USE_SYSTEM_SUITESPARSE=1` will then use the SuiteSparse that has been copied into Julia's directory, but will not build a new SuiteSparse library from scratch.
```

Comment:
I have no idea why this script was needed, but it still exists. I'd say we should either remove the script and its documentation, or keep both?

Otherwise, looks good to me.

Reply:
I'll remove the script. Not necessary any more.

Reply:
Clearly, it was necessary for the mac travis!

Reply:
So is it also needed for Mac users in general?

Reply:
No, it's just for those who want to package Julia with their system package manager, but can't just fix their system package manager (aka pretty much just our CI). Oh, and also, pretty much just limited to macOS (the default compiler flags on most linux system break this script).